src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public abstract class Figure implements FigureDraw, FigureArea {
+    private Colors color;
+
+    public void setColor(Colors color) {
+        this.color = color;
+    }
+
+    public Colors getColor() {
+        return color;
+    }
+
+    @Override
+    public abstract double getArea();
+
+    @Override
+    public abstract void draw();

```

Comment:
do you need to override them here?

Reply:
U don't need to override abstract methods from interface in abstract class, if you don't want to create default implementation. Since, you don't need to - just remove.